examples/tensorboard/Logging_DLPy_Scalars_TensorBoard .ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
seems like there is a white space in the name. please remove it

Reply:
Fixed